### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Ibizugbe**
In this PR, the current milestone of the project has been completed in line with the specified requirements.
- no linter errors.
- the project is professionally documented.
- best practices were adopted when using JavaScript.
- the project has been styled, with reference to the given design.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,5 +1,151 @@
-ul {
-  padding: 0;
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,900&display=swap");
+
```
Comment (mirouhml) [inline src/App.css]: - [ ] Kindly use the correct font families for the website, there are different ones on different parts of the website, please double-check the design provided and add the correct font families. 😁

File: src/components/Form.js [hunk 2]
```diff
@@ -31,29 +31,34 @@ function Form() {
...
+        className="inputs"
         type="text"
         name="category"
         value={formValue.category}
         onChange={changeState}
         placeholder="category"
       />
-      <input type="submit" value="Add Book" />
+      <input type="submit" value="Add Book" className="btn btn-primary" />
     </form>
```
Comment (mirouhml) [inline src/components/Form.js:62]: - [ ] Kindly fix the styling for this section it should look like the design provided, like this:
image

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @Ibizugbe, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi @Ibizugbe, 👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥💯💯



## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality ✅
📌 Good commit messages ✅
📌Professional README file. ✅
📌 Great design 🔥🔥

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**